sys/include/xtimer.h

```diff
@@ -184,120 +189,129 @@ static inline void xtimer_tsleep64(xtimer_ticks64_t ticks);
  *
  * This function will spin-block, so only use it *very* short periods.
  *
- * @param[in] ticks  the number of xtimer ticks the thread should spin for
+ * @param[in] ticks   the number of xtimer ticks the thread should spin for
  */
 static inline void xtimer_spin(xtimer_ticks32_t ticks);
 
 /**
  * @brief will cause the calling thread to be suspended until the absolute
- * time (@p last_wakeup + @p period).
+ * time (@p last_wakeup_ticks + @p period_us [in ticks]).
  *
- * When the function returns, @p last_wakeup is set to
- * (@p last_wakeup + @p period).
+ * When the function returns, @p last_wakeup_ticks is set to
+ * (@p last_wakeup_ticks + @p period_us [in ticks]).
  *
  * This function can be used to create periodic wakeups.
- * @c last_wakeup should be set to xtimer_now() before first call of the
+ * @c last_wakeup_ticks should be set to xtimer_now() before first call of the
  * function.
  *
- * If the result of (@p last_wakeup + @p period) would be in the past, the function
- * sets @p last_wakeup to @p last_wakeup + @p period and returns immediately.
+ * If the result of (@p last_wakeup_ticks + @p period_us [in ticks]) would be in
+ * the past, the function sets @p last_wakeup_ticks to
+ * @p last_wakeup_ticks + @p period_us [in ticks] and returns immediately.
  *
- * @param[in] last_wakeup   base time stamp for the wakeup
- * @param[in] period        time in microseconds that will be added to last_wakeup
+ * @param[in] last_wakeup_ticks   base time stamp for the wakeup
+ * @param[in] period_us           time in microseconds that will be added to
+ *                                last_wakeup
  */
-static inline void xtimer_periodic_wakeup(xtimer_ticks32_t *last_wakeup, uint32_t period);
+static inline void xtimer_periodic_wakeup(xtimer_ticks32_t *last_wakeup_ticks,
+                                          uint32_t period_us);
 
 /**
  * @brief Set a timer that sends a message
  *
- * This function sets a timer that will send a message @p offset ticks
- * from now.
+ * This function sets a timer that will send a message @p offset_us
+ * microseconds from now.
  *
- * The mesage struct specified by msg parameter will not be copied, e.g., it
+ * The message struct specified by msg parameter will not be copied, e.g., it
```

Comment:
grammar, feel free to fix or leave since it's not part of your change:
e.g. ("exempli gratia", for example) -> i.e. ("id est", that is, in other words)